crates/storage/provider/src/block.rs

```diff
@@ -117,10 +117,11 @@ pub struct ChainInfo {
 /// Inserts blocks data to [tables::CanonicalHeaders], [tables::Headers], [tables::HeaderNumbers],
 /// and transactions data to [tables::TxSenders], [tables::Transactions],
 /// [tables::CumulativeTxCount] and [tables::BlockBodies]
-pub fn insert_canonical_block<'a, TX: DbTxMut<'a> + DbTx<'a>>(
```

Comment:
Was it a mistake to call it canonical before?

Reply:
insert_canonical_block function is still present below. This is mostly used for tests